### PR TITLE
Credits with card details

### PR DESCRIPTION
**Pull request by ct-clearhaus**

### CODE REVIEW

File: source/index.md [hunk 1]
```diff
@@ -713,13 +713,53 @@ No parameters are needed to make a new void transaction.
...
 #### Parameters
 
 ````
-POST https://gateway.clearhaus.com/cards/:id/credits
+POST https://gateway.clearhaus.com/credits
+````
+
+<dl class="dl-horizontal">
+  <dt>amount</dt>
+  <dd>[1-9][0-9]{0,9} <br />
```
Comment (mt-clearhaus) [inline source/index.md]: We use `[0-9]{1,10}` for amount for `/authorizations` now :slightly_smiling_face:

Comment (mt-clearhaus) [reply source/index.md]: Of course, we will have to adapt https://github.com/clearhaus/klovedal/blob/5f097d212b9f5d87724ab5ceb0c67f2b13e8f7b4/lib/klovedal/request/credit.rb#L8

File: source/index.md [hunk 2]
```diff
@@ -713,13 +713,53 @@ No parameters are needed to make a new void transaction.
...
+    A reference to an external object, such as an order number.
+  </dd>
+  <dt>card[pan]</dt>
+  <dd>[0-9]{12,19} <br /> Primary account number of card to charge.</dd>
+  <dt>card[expire_month]</dt>
+  <dd>[0-9]{2} <br /> Expiry month of card to charge.</dd>
+  <dt>card[expire_year]</dt>
+  <dd>20[0-9]{2} <br /> Expiry year of card to charge.</dd>
+  <dt>card[csc]</dt>
+  <dd>[0-9]{3} <br /> <i>Optional.</i> <br /> Card Security Code.</dd>
```
Comment (mt-clearhaus) [inline source/index.md]: We need to decide if we want to offer `card[csc]` or not.

File: source/index.md (hunk 1 above)
Comment (ct-clearhaus) [reply source/index.md]: "0-credits" should not be allowed; 0-auths are allowed.

File: source/index.md (hunk 2 above)
Comment (ct-clearhaus) [reply source/index.md]: Indeed. We've allowed it so far; I guess we can check if we ever processed credits with CSC?

File: source/index.md (hunk 1 above)
Comment (mt-clearhaus) [reply source/index.md]: Do'h. Good point.

File: source/index.md (hunk 2 above)
Comment (mt-clearhaus) [reply source/index.md]: We have not processed credits with CSC so far.

Comment (ct-clearhaus) [reply source/index.md]: Let's kill it.

Comment (ct-clearhaus) [reply source/index.md]: Uhm, or, wait. We should perhaps consider how many `POST /cards?card[csc]=...` was done "just before" the accompanying `POST /cards/:id/credits` ...

Comment (ct-clearhaus) [reply source/index.md]: Let's keep it for now and remove later if never used.

**Comment by ct-clearhaus** [comment]
@mt-clearhaus Should we include another entry in the "Changes" section?

File: source/index.md [hunk 3]
```diff
@@ -1015,6 +1015,12 @@ https://gateway.clearhaus.com/account
 
 ## Changes
 
+### Add credits resource
+
+Starting 2018-xx-xxTxx:xx:xx+00:00 the `/credits` resource has been added,
+enabling creation of credits without card tokenization. This is an essential
+addition, as card tokenization is deprecated.
```
Comment (ct-clearhaus) [inline source/index.md]: Uhm, I think the more important note is that `/cards/:id/credits` will also be deprecated — and what the deadline for this functionality is.

File: source/index.md [hunk 4]
```diff
@@ -975,6 +1015,13 @@ https://gateway.clearhaus.com/account
 
 ## Changes
 
+### Add credits resource
+
+Starting 2018-xx-xxTxx:xx:xx+00:00 the `/credits` resource has been added,
```
Comment (ct-clearhaus) [inline source/index.md]: IMO: `s/Starting 2018-xx-xxTxx:xx:xx+00:00 the/The/` and then we just don't deploy this PR before the implementation is in production.

We shouldn't forget also to communicate this explicitly to partners (that do credits).

Comment (mt-clearhaus) [reply source/index.md]: We have starting dates for a few other change announcements, so I will keep it. I acknowledge that a date is partly deducible from this PR (which is public), however, I think a changelog needs versions, dates or both in order to be usable on its own.

**Comment by mt-clearhaus** [comment]
Deployed to http://docs.gateway.clearhaus.com :heavy_check_mark:

**Pull request merged**